server/static/hardware_list.jsx

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const e = React.createElement;
+
+class HardwareList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { hardware: [] };
+    }
+
+    componentDidMount() {
```

Comment:
What does this do?

Reply:
This just polls the server for changes in the hardware status.